java-test-samples/java-appsync-sam/README.md

```diff
@@ -0,0 +1,217 @@
+# java-appsync-sam
```

Comment:
I recommend adding a paragraph to describe the purpose of the app and what it does. Would help to provide more context. Perhaps start with an Overview section?

Reply:
Or maybe it's a section that describes the **System Under Test**?